relevanceai/operations_new/cluster/ops.py

```diff
@@ -19,8 +20,8 @@ def __init__(
         dataset_id: str,
         vector_fields: list,
         alias: str,
-        cluster_field: str,
-        *args,
+        cluster_field: str = "_cluster_",
```

Comment:
why do we do this, we have so much other tech that relies off this being the `cluster_field`. why do we give users the opportunity to change this?

Reply:
They shouldn't be changing or touching it. This is more for us if we ever need to.

Reply:
ok